package.json

```diff
@@ -25,20 +28,34 @@
   },
   "homepage": "https://github.com/oliverviljamaa/angularjs-test#readme",
   "devDependencies": {
+    "angular": "^1.7.2",
+    "angular-mocks": "^1.7.2",
     "babel-core": "^6.26.3",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
     "babel-preset-env": "^1.7.0",
     "eslint": "^5.2.0",
     "eslint-config-airbnb-base": "^13.0.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-import": "^2.13.0",
     "eslint-plugin-prettier": "^2.6.2",
     "husky": "^0.14.3",
+    "jest": "^23.4.2",
```

Comment:
This component is probs not backwards compatible with Jasmine anymore is it?

Reply:
It is, it doesn't make any assumptions on the test framework/runner, I'll update the README, thanks!